magefile.go

```diff
@@ -201,6 +208,18 @@ func customizePackaging() {
 			args.Spec.ReplaceFile("/usr/share/{{.BeatName}}/README.md", readmeTemplate)
 			args.Spec.Files["/usr/share/{{.BeatName}}/"+ingestTarget] = ingest
 
+			// update config file Owner
+			pf := args.Spec.Files["/etc/{{.BeatName}}/{{.BeatName}}.yml"]
+			pf.Owner = mage.BeatUser
+			args.Spec.Files["/etc/{{.BeatName}}/{{.BeatName}}.yml"] = pf
+
+			args.Spec.Files["/var/lib/{{.BeatName}}"] = mage.PackageFile{Mode: 0750, Source: emptyDir, Owner: mage.BeatUser}
+			args.Spec.Files["/var/log/{{.BeatName}}"] = mage.PackageFile{Mode: 0750, Source: emptyDir, Owner: mage.BeatUser}
```

Comment:
for some reason `var/lib/apm-server` is owned by `apm-server` but `var/log/apm-server` is owned by `root`. any idea why? i don't see anything wrong with this

Reply:
`/var/log/apm-server` should be owned by apm-server so that log files can be created/rotated.  I see them owned by `apm-server` in testing, how did you try it out?

Reply:
this is what I do:

~/go/src/github.com/elastic/apm-server/build/distributions$ sudo dpkg -i apm-server-oss-7.0.0-amd64.deb

~$ sudo -u apm-server apm-server setup
2019-01-24 16:58:47.967389676 +0100 CET m=+0.019150500 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.968451011 +0100 CET m=+0.020211803 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.96853076 +0100 CET m=+0.020291551 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.968585792 +0100 CET m=+0.020346582 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.968656242 +0100 CET m=+0.020417043 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.970918727 +0100 CET m=+0.022679552 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.971314113 +0100 CET m=+0.023074902 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.971355146 +0100 CET m=+0.023115939 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.971962761 +0100 CET m=+0.023723561 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.972321619 +0100 CET m=+0.024082420 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.972435703 +0100 CET m=+0.024196503 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.972570193 +0100 CET m=+0.024331018 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.975554538 +0100 CET m=+0.027315331 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.977419839 +0100 CET m=+0.029180635 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.97747411 +0100 CET m=+0.029234906 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
Loaded index template
Loading dashboards (Kibana must be running and reachable)
2019-01-24 16:58:47.977678306 +0100 CET m=+0.029439097 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.98763534 +0100 CET m=+0.039396128 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
2019-01-24 16:58:47.9877635 +0100 CET m=+0.039524324 write error: failed to open new file: open /var/log/apm-server/apm-server: permission denied
Skipping loading dashboards, No dashboards to import. Please make sure the /usr/share/apm-server/kibana/7 directory contains a dashboard directory.
~$ sudo ls -la /var/log/apm-server/
total 8
drwxr-xr-x  2 root root   4096 Jan 24 13:53 .
drwxrwxr-x 15 root syslog 4096 Jan 24 16:36 ..


I don't really get how that happens :/

Reply:
that is on fresh install, so `var/log/apm-server` doesn't exist before

Reply:
I will try to reproduce in a VM, maybe testing under docker is insufficient.